### PR TITLE
Ahmad Design Changes

### PR DESCRIPTION
**Pull request by sabeurthabti**
- [x] Add timing of the current surah time
- [x] Progress bar under the currently playing surah

screen shot 2016-12-18 at 22 49 30

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33010

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33011

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33012

File: src/components/Audioplayer/Track/style.scss [hunk 1]
```diff
@@ -27,3 +26,7 @@
     transition: height 0.5s;
   }
 }
+
+.simple:after {
+    display: none;
+  }
```
Comment (mmahalwy) [inline src/components/Audioplayer/Track/style.scss]: space

File: src/containers/Qari/index.js [hunk 2]
```diff
@@ -38,7 +40,7 @@ class Qaris extends Component {
   }
 
   render() {
-    const { surahs, qari, files, currentSurah, Playing, shouldRandom, currentQari} = this.props;
+    const { surahs, qari, files, currentSurah, isPlaying, shouldRandom, currentQari, currentTime, progress} = this.props;
```
Comment (mmahalwy) [inline src/containers/Qari/index.js]: space after `progress`

File: src/containers/Qari/index.js [hunk 3]
```diff
@@ -50,6 +52,11 @@ class Qaris extends Component {
     };
 
     const description = qari.description ? qari.description : '';
+    const currentSurahTime = (surah) => {
+      return (surah.id === currentSurah.id) ? `${formatSeconds(currentTime)} / ` : '';
+    };
+
+    const progressBarStyle = {'position': 'absolute', bottom: '-5px', height: '2px'};
```
Comment (mmahalwy) [inline src/containers/Qari/index.js]: position in quotes?

**Comment by sabeurthabti** [comment]
@mmahalwy thank you

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33013

**Pull request merged**